build/PlatformInstaller.targets

```diff
@@ -231,26 +151,26 @@
     <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_BuildDockerImage" Properties="Image=$(Image)" />
 
     <!-- Common installer parameters -->
-    <JsonPeek Query="$.short_description" JsonInputPath="$(RSDebConfigFile)">
-      <Output TaskParameter="Result" ItemName="_RSInstallerSummary" />
+    <JsonPeek Query="$.short_description" JsonInputPath="$(SFDebConfigFile)">
```

Comment:
This is rather ugly, I'm going to create a props file that contains most of this information and populate config files instead. This way the truth lies in msbuild files instead of json files.